packages/website/blog/2024-04-30-announcing-typescript-eslint-v8-beta.md

```diff
@@ -0,0 +1,261 @@
+
+1. Remove all your rules configurations
+2. Extend from the preset configs that make sense for you
+3. Run ESLint on your project
+4. In your ESLint configuration, turn off any rules creating errors that don't make sense for your project - with comments explaining why
+5. In your ESLint configuration and/or with inline `eslint-disable` comments, turn off any rules creating too many errors for you to fix - with _"TODO"_ comments linking to tracking issues/tickets to re-enable them
+
+:::
+
+### Rule Breaking Changes
+
+Several rules were changed in significant enough ways to be considered breaking changes:
+
+- ⏳ [Rules: Deprecate prefer-ts-expect-error in favor of ban-ts-comment](https://github.com/typescript-eslint/typescript-eslint/issues/8333)
+- ⏳ [chore(eslint-plugin): deprecate no-var-requires in favor of no-require-imports](https://github.com/typescript-eslint/typescript-eslint/pull/8334)
+- [feat(eslint-plugin): deprecate no-throw-literal and add a renamed only-throw-error](https://github.com/typescript-eslint/typescript-eslint/pull/8701)
+- ⏳ [fix(eslint-plugin): [no-useless-template-literals] rename to no-useless-template-expression](https://github.com/typescript-eslint/typescript-eslint/pull/8821)
+- [feat(eslint-plugin): deprecate no-loss-of-precision extension rule](https://github.com/typescript-eslint/typescript-eslint/pull/8832)
+- [feat(eslint-plugin): remove formatting/layout rules](https://github.com/typescript-eslint/typescript-eslint/pull/8833)
+- [feat(eslint-plugin): [prefer-nullish-coalescing] change ignoreConditionalTests default to true](https://github.com/typescript-eslint/typescript-eslint/pull/8872)
+- [feat(eslint-plugin): [no-unused-vars] align catch behavior to ESLint 9](https://github.com/typescript-eslint/typescript-eslint/pull/8971)
+- ⏳ [feat(eslint-plugin): split no-empty-object-type rule out from ban-types rule](https://github.com/typescript-eslint/typescript-eslint/pull/8977)
+
+### Tooling Breaking Changes
+
+- ⏳ [Enhancement: Error if configuration options aren't provided as expected](https://github.com/typescript-eslint/typescript-eslint/issues/6403)
+- [fix(typescript-estree): enable dot globs for project by default](https://github.com/typescript-eslint/typescript-eslint/pull/8818)
+- [feat(typescript-estree): remove slow deprecated and isolated programs](https://github.com/typescript-eslint/typescript-eslint/pull/8834)
+- [feat(typescript-estree): rename automaticSingleRunInference to disallowAutomaticSingleRunInference](https://github.com/typescript-eslint/typescript-eslint/pull/8922)
+- [chore: bump minimum versions for v8](https://github.com/typescript-eslint/typescript-eslint/pull/8973)
+  - ESLint support range was changed from `^8.56.0` to `^8.57.0`
+  - Node.js support range was changed from `^18.18.0 || >=20.0.0` to `^18.18.0 || ^20.9.0 || >=21.1.0`
+  - TypeScript support range was changed from `>=4.7.4 <5.5.0` to `>=4.8.4 <5.5.0`
```

Comment:
one thing I think would be good to do here is expand on each dot point with a concrete user action.
Otherwise each user has to either try and parse meaning out of the PR title or click through and parse the PR itself.

Something like

- fix(typescript-estree): enable dot globs for project by default
    - This will cause any `parserOptions.project` globs to match dot directories. If you don't want to match them then you must foo the bar.
- feat(typescript-estree): rename automaticSingleRunInference to disallowAutomaticSingleRunInference
    - We've updated the default to be an opt-out - meaning you no longer need to enable it:
    ```diff
     parserOptions: {
    -  automaticSingleRunInference: true,
       // ...
     },
    ```

etc